src/services/scheduling/jobs/process-batches-job.js

```diff
@@ -7,6 +7,7 @@ import TaskProcessor from '../batch-task-processors/task-processor.js';
 const logger = new Logger(import.meta.url);
 
 const IDLE_POLL_INTERVAL_IN_MS = 10000;
+const BUSY_POLL_INTERVAL_IN_MS = 0;
```

Comment:
Micro thing, let's call these `DEFAULT_IDLE_POLL_INTERVAL_IN_MS` and `DEFAULT_BUSY_POLL_INTERVAL_IN_MS`.